src/retasc/templates/extensions/dates.py

```diff
@@ -28,6 +36,8 @@ def weeks(value: int) -> timedelta:
 FILTERS = {
     "days": days,
     "weeks": weeks,
+    "day": day,
+    "week": week,
```

Comment:
We do not need new function:

```suggestion
    "day": days,
    "week": weeks,
```